spec/requests/api/v1/medical_shifts_request_spec.rb

```diff
@@ -322,4 +322,37 @@
       end
     end
   end
+
+  describe "DELETE /api/v1/medical_shifts/:id" do
+    let(:user) { create(:user) }
+    let(:medical_shift) { create(:medical_shift, user_id: user.id) }
+
+    context "when user is authenticated" do
```

Comment:
is the not found scenario being handled by api_controller? 
Is this the reason why we don't have specs for it?

Reply:
It was my mistake. I take some examples as guide and ended up that I forgot some tests 43daa84a704a3fa90e91cf6d9ad30deb871b9adc

Reply:
The commit  https://github.com/espoo-dev/med_system_backend/commit/43daa84a704a3fa90e91cf6d9ad30deb871b9adc does not add specs for not_found

Reply:
This ee8ad3ca88bfef2681bbb59c1d2f8447ed08383c is the correct hash commit. sorry